rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -221,6 +228,7 @@ void SetupResumeAcceptor::close(
     yarpl::Reference<FrameTransport> tport,
     folly::exception_wrapper e) {
   DCHECK(eventBase_->isInEventBaseThread());
+  DCHECK(tport);
```

Comment:
I have the reverse question in here. What was the reason of performing DCHECK in here instead of if(..) return?